### ISSUE TITLE
[Accessibility] Support task completion/failure audio cues in Output

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#176521]
CC @meganrogge 

Many extensions, such as [microsoft/vscode-arduino](https://github.com/microsoft/vscode-arduino), use Output view and display compile results like below:

![image](https://user-images.githubusercontent.com/19754711/223737392-4a0cfa2f-1c44-4c3e-b24b-5c54b6b87cd7.png)

Could you please support task completion and failure audio cues in output view as well? Currently, it is very tedious to read through the results line by line.

**Comment by meganrogge** [comment]
Seems like a reasonable request, though I'm not sure what would be involved here. @sandy081 do you think this would be possible?

**Comment by sandy081** [comment]
Output view uses editor in the background. @alexdima Does the output view has to do anything special to enable accessibility features ?

**Comment by meganrogge** [comment]
@jooyoungseo , to confirm, are you requesting that the audio gets played when you run the command depending on success or failure?

if so, I think we'd need extension API to support this

**Comment by jooyoungseo** [comment]
@meganrogge -- Yes, that's exactly correct.

**Comment by meganrogge** [comment]
I spoke with @isidorn and he brought up some great points:

1. Could the arduino extension use a task for this so that no additional steps are needed for the task failure/success audio cues to happen?
2. If not, the arduino extension should consider playing our task completion / failure sounds within the extension instead of API. An example of an extension that plays audio cues is Liveshare.

**Comment by meganrogge** [comment]
@benmcmorran thanks for your reply. Is either of those something you plan to do in the near future? It seems like an important feature
